src/clarity-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -373,6 +373,11 @@
             vertical-align: bottom;
         }
 
+        .datagrid-signpost-trigger {
+            height: $clr_baselineRem_1_5 - 1;
+            padding: 0 12px 0;
+        }
```

Comment:
How is the vertical alignment of the text with respect to other cells preserved? 

Based on the screenshots I'm going to trust that it works, but I have to admit I'm surprised.